ui/src/components/ConfigDetailsMenu/ConfigDetailsMenu.tsx

```diff
@@ -98,6 +100,12 @@ export const ConfigDetailsMenu: React.FC<ConfigDetailsMenuProps> = ({
             </ListItemIcon>
             <ListItemText>Duplicate current version</ListItemText>
           </MenuItem>
+          <MenuItem onClick={() => setAdvancedDialogOpen(true)}>
+            <ListItemIcon>
+              <SettingsIcon width="20px" stroke={"currentColor"} />
+            </ListItemIcon>
+            <ListItemText>Advanced Config</ListItemText>
```

Comment:
I think we're just using "Configuration" everywhere now.
```suggestion
            <ListItemText>Advanced Configuration</ListItemText>
```